<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="nodesListWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>261</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="debugSendText">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send text </string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="debugTextBrowser">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="debugSendLineEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
